nautobot_chatops_ansible/worker.py

```diff
@@ -88,16 +91,16 @@ def get_inventory(dispatcher, inventory, group):
 
     if not group:
         data = tower.get_tower_inventory_groups(inventory=inventory)
+        LOGGER.debug("Data result: %s", data)
         dispatcher.prompt_from_menu(
-            f"ansible get-inventory {inventory}",
+            f'ansible get-inventory "{inventory}"',
             "Select inventory group",
-            [(entry["name"], entry["name"]) for entry in data["results"]],
+            [(f"{entry['name']}", f"{entry['name']}") for entry in data["results"]],
```

Comment:
```suggestion
            [(entry["name"], entry["name"]) for entry in data["results"]],
```
This is definitely readable but a little confusing at first glance.  What about something like the above?